app/javascript/components/HomePageContent.js

```diff
@@ -29,9 +30,9 @@ const LinkableSearchTabs = function(props) {
   const showGenesTab = location.pathname.startsWith('/single_cell/app/genes')
   return (
     <div>
-      <nav className="nav search-links">
+      <nav className="nav search-links" data-tablist-name={'Search tabs'} role="tablist">
```

Comment:
The other tablist names omit "tabs", so let's so here as well for consistency:

```suggestion
      <nav className="nav search-links" data-tablist-name={'Search'} role="tablist">
```

Reply:
9adbd77e